tests/helpers/test_sun.py

```diff
@@ -184,10 +190,10 @@ def test_norway_in_june(hass):
     print(sun.get_astral_event_date(hass, SUN_EVENT_SUNSET, datetime(2017, 7, 26)))
 
     assert sun.get_astral_event_next(hass, SUN_EVENT_SUNRISE, june) == datetime(
-        2016, 7, 25, 23, 23, 39, tzinfo=dt_util.UTC
+        2016, 7, 24, 22, 59, 45, 689645, tzinfo=dt_util.UTC
```

Comment:
Why does this change?

Reply:
(copy-paste of relevant information from above)
The new Astral both produces different times and provides times down to the MS level. To deal with that the tests need to be re-written. Tests were written to match old Astrals less accurate data.

I got confirmation here:
https://github.com/home-assistant/core/pull/48282#issuecomment-808855672